logos/logos-workernode/logos_worker_node/runtime.py

```
@@ -158,10 +161,63 @@ async def build_runtime_status(app: FastAPI) -> WorkerRuntimeStatus:
+                if _name in _file_state:
+                    _prof["calibration_unsupported"] = True
+                    _prof["calibration_unsupported_reason"] = _file_state[_name].reason_code
+                elif _prof.get("calibration_unsupported"):
+                    # File no longer mentions this model — clear the flag
+                    # both on the wire and in the in-memory registry so the
+                    # next persist drops the stale value.
+                    _prof["calibration_unsupported"] = None
+                    _prof["calibration_unsupported_reason"] = None
+                    app.state.model_profiles.mark_calibration_unsupported(_name, False)
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)